README.md

```diff
@@ -2,478 +2,34 @@
   <img src="images/irnas-logo.png" alt="irnas-logo" width=400 ><br><br>
 </p>
```

Comment:
maybe make a logo a bit smaller?
less scrolling needed

Reply:
I agree that is too big, I willl actually change the image so that is more wide and less tall.